content/en/logs/guide/collect-google-cloud-logs.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Collect Google Cloud logs
+kind: documentation
+further_reading:
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn how to explore your logs"
+- link: "/logs/explorer/#visualize"
+  tag: "Documentation"
+  text: "Perform Log Analytics"
+- link: "/logs/log_configuration/processors"
+  tag: "Documentation"
+  text: "Learn how to process your logs"
+---
+
+## Overview
+
+<div class="alert alert-warning">
+Pulling logs from a Google Cloud Pub/Sub topic with the Datadog Dataflow template is in beta. Datadog recommends using this approach for compression and batching of events forwarded to Datadog. Use the <a href="https://cloud.google.com/products/calculator" target="_blank">Google Cloud Pricing Calculator</a> to calculate any additional costs you may incur.
+</div>
+
+You can collect your Google Cloud logs in Datadog by pushing them through a subscription to a [Google Cloud Pub/Sub][10] topic or by pulling them from the topic with the [Datadog Dataflow template][11] (beta).
+
+For applications running in GCE or GKE, the [Datadog Agent][13] can be used to collect logs locally. GCP service logs are collected with Google Cloud Logging and sent to a Cloud Pub/Sub with a HTTP Push forwarder. 
+
+## Setup
+
+### Prerequisites
+
+1. The [Google Cloud platform integration][1] is successfully installed.
+2. Both methods for sending Google Cloud logs to Datadog require a subscription to a Google Cloud Pub/Sub topic. 
+   - If you are using the [Subscription to Google Cloud Pub/Sub](#subscription-to-google-cloud-pubsub) method, configure the subscription with a **Push** delivery type. 
+   - If you are using the [Datadog Dataflow template](#datadog-dataflow-template) method (beta), configure the subscription with a **Pull** delivery type.
+
+#### Create a Cloud Pub/Sub
+
+1. Go to the [Cloud Pub/Sub console][2] and create a new topic.
+
+    {{< img src="integrations/google_cloud_platform/create_a_topic.png" alt="Create a topic" style="width:80%;">}}
+
+2. Give that topic an explicit name such as `export-logs-to-datadog` and click _Create_.
+
+**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][3]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [the Monitor the Log Forwarding section](#monitor-the-log-forwarding) for information on how to set up a monitor to be automatically notified if you get close to those limits.
+
+### Subscription to Google Cloud Pub/Sub
```

Comment:
Can we please put this in a separate page and instead link to that page from the main Google Cloud Platform page. The thinking is that if we make this less visible, new customers will adopt the Dataflow method, which is the path we are recommending for all users.

Reply:
Certainly, but just to make sure I'm understanding correctly:
- The new Dataflow template approach should be featured on the main Google Cloud Platform page
- The older Push subscription method should have its own guide that's linked to from the main Google Cloud page
Is that correct?

Reply:
@buraizu that's right. sorry for the delay on getting back here.